toolkit/committee-selection/authority-selection-inherents/src/tests.rs

```diff
@@ -341,7 +341,7 @@ fn ariadne_4_to_7_test() {
 	committee_names.sort();
 	// Each permissioned has 1 guaranteed. henry, ida, james and kim as well. There are 3 randomly selected: eve, ferdie and james.
 	let expected_committee_names = vec![
-		"alice", "bob", "charlie", "dave", "eve", "ferdie", "henry", "ida", "james", "james", "kim",
+		"alice", "bob", "charlie", "dave", "eve", "eve", "ferdie", "henry", "ida", "james", "kim",
```

Comment:
For `ariadne_4_to_7_test`, it is selecting twice `eve` instead of `james`: 
```
before: ["alice", "bob", "charlie", "dave", "eve", "ferdie", "henry", "ida", "james", "james", "kim"]
after: ["alice", "bob", "charlie", "dave", "eve", "eve", "ferdie", "henry", "ida", "james", "kim"] 
```
It is associated with upgrade of `rand_chacha`. Is it correct?

Reply:
As seen, the selection algorithm for the same inputs now have different outputs. This should not be a problem, because whole chain will start using it after runtime upgrade.

Please do something with the comment two lines above. I think I shouldn't have enumerated these 3 randomly  selected identifiers.